src/_includes/layouts/homepage.html

```diff
@@ -6,35 +6,43 @@
     <link rel="stylesheet" href="css/reset.css" />
     <link rel="stylesheet" href="css/styles.css" />
     <link rel="stylesheet" href="/css/navbar.css" />
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@900&family=Source+Code+Pro:ital,wght@0,400;0,600;1,500;1,600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/css/homepage.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@900&family=Source+Code+Pro:ital,wght@0,400;0,600;1,500;1,600&display=swap"
+      rel="stylesheet"
+    />
     <title>Lunch.dev Community Calendar</title>
   </head>
   <body data-typeface="system-ui">
-	{% include partials/header.html %}
-    <main data-layout="centered-single-column">
-      <h1>Lunch.dev Community Calendar</h1>
+    {% include partials/header.html %}
 
-      <h2 id="upcoming-events-section">Upcoming Events</h2>
-      <ul aria-labelledby="upcoming-events-section">
+    <main data-layout="centered-single-column">
+      <h2>Upcoming Events</h2>
+      <section id="upcoming-events-section" class="events-section">
         {% for event in collections.upcomingEvents %} {{ event | log }}
-        <li>
-          <a href="{{ event.url }}"> {{ event.data.title }} </a>
+        <article class="event-item">
```

Comment:
I love the markup here! You've done a great job! I wanted to propose a bit of [ARIA](https://benmyers.dev/blog/aria) that might help make this even better for screenreader users. Specifically, if we use ARIA to _label_ these articles, it'll make it much easier for screenreader users to hop from `article` to `article`.

[We can use the `h3`s as our label](https://benmyers.dev/blog/aria-labels-and-descriptions/#aria-labelledby) — here's the flow I'm thinking:

1. Give the `h3` an `id` - we could use [Eleventy's built-in `slug` filter](https://www.11ty.dev/docs/filters/slug/) to generate it from the `title` frontmatter.
2. Then, we set the `article`'s `aria-labelledby` attribute to the `h3`'s `id`.